code/socialDistribution/dispatchers.py

```diff
@@ -1,41 +1,76 @@
 from typing import List
-from typing_extensions import ParamSpecArgs
 import requests
 import json
 
-from .models import LocalPost, LocalAuthor
+from .models import LocalPost, Author, LocalAuthor
 
-def send(post, follower):
-    author_inbox = follower.url + "/inbox"
+
+def dispatch_post(post: LocalPost, recipients: List[LocalAuthor] = None):
+    """ Sends a post to the inbox of all followers who have permission to view the post.
+
+    Parameters:
+        post (LocalPost): the post to be sent out
+        recipients (QuerySet of LocalAuthor): the list of authors to receive the post if the post is private
+    """
+
+    if post.visibility == LocalPost.Visibility.PUBLIC:
+        # send posts to all followers
+        for follower in post.author.followers.all():
```

Comment:
Should `post.author.followers.all()` be `post.author.follows.all()`? I'm getting an error when I make a public post: `AttributeError at /app/author/1/posts/
'LocalAuthor' object has no attribute 'followers'
line 18, in dispatch_post`

Reply:
Yep. You're right, I missed it. Thanks for catching that. Will make a quick fix.